src/visualization/recipes_makie.jl

```diff
@@ -22,17 +28,13 @@ Inputs:
 !!! warning "Experimental implementation"
     This is an experimental feature and may change in future releases.
 """
-function iplot(u, mesh::UnstructuredMesh2D, equations, solver, cache;
-               solution_variables=nothing, nvisnodes=2*nnodes(solver), variable_to_plot_in=1,
-               colormap = default_Makie_colormap())
-
-  pd = PlotData2D(u, mesh, equations, solver, cache;
-                  solution_variables=solution_variables, nvisnodes=nvisnodes)
-
+# Enables `iplot(PlotData2D(sol))`. Note that `nvisnodes=nothing` since
+# `PlotData2DTriangulated` doesn't contain any information about the solver.
+function iplot(pd::PlotData2DTriangulated;
+               plot_mesh=true, show_axis=false, colormap=default_Makie_colormap(),
+               nvisnodes=nothing, variable_to_plot_in=1)
```

Comment:
Do I understand it correctly that `nvisnodes` has no effect here? Wouldn't it in this case be better to omit this argument entirely, or is it still needed? If this is really only about consistency vs. POLA, I'd favor the solution that does not surprise the user (by having an argument without effect)

Reply:
That's a good question. I'm not sure why it appears to be unused, let me check.

Reply:
OK, so what happened?

Reply:
It was unused. It gets passed into the version intended for dispatch on `sol` or `u, mesh_equations_solver_cache(semi)...` but not when the primary argument is a `PlotData2DTriangulated`.

Reply:
I've removed the keyword argument wherever it wasn't used.

Reply:
:+1: